doc/setup-mac.md

```diff
@@ -49,32 +67,11 @@ gem install bundler
 # (from the bottlenose directory checked out from git)
 bundle install
 ```
-### Capybara-webkit
-
-**Prerequisite: Xcode (between 9.4 - 10.1).** Newer versions of Xcode will not work and you will have to downgrade! To do this follow these [instructions] (https://medium.com/@tseboho/how-to-downgrade-xcode-4359df5158d5)
-
-**NOTE:** 
 
-Qt 5.5 is the last version of Qt that capybara-webkit will support. The Qt project has dropped the WebKit bindings from binary releases in 5.6. Make sure that you have Xcode installed (between 9.4 - 10.1). Under Xcode preferences locations, make sure there is a version set. Qt 5.5 was removed from homebrew in newer commits. The previous/parent commit was **9ba3d6e**. So, to be able to install Qt 5.5 with homebrew checkout the old commit first:
+### Javascript
 
 ```
-brew update
-cd $( brew --prefix )/Homebrew/Library/Taps/homebrew/homebrew-core
-git checkout 9ba3d6ef8891e5c15dbdc9333f857b13711d4e97 Formula/qt@5.5.rb
-brew install qt@5.5
-```
-**NOTE:**  
-
-- If you get Error fatal: reference is not a tree: 9ba3d6ef8891e5c15dbdc9333f857b13711d4e97, use git fetch --unshallow to complete git history.
-- If you get Error: qt@5.5: unknown version :mountain_lion or :mojave, comment out line #25 in Formula/qt@5.5.rb
-
-The Homebrew formula for qt@5.5 is keg only which means binaries like qmake will not be symlinked into your /usr/local/bin directory and therefore will not be available for capybara-webkit.
-```
-# Then add to your shell configuration file:
-echo 'export PATH="$(brew --prefix qt@5.5)/bin:$PATH"' >> ~/.bashrc	
-
-# Now you can install capybara-webkit
- gem install capybara-webkit
+npm install
```

Comment:
We've got a `yarn.lock` file in the project code, so I'm thinking we might want to swap this out with some `yarn` instructions (or alternatively replace that with a `package-lock.json` file instead).

Not a huge deal, just thinking it would be good for consistency

Reply:
Nevermind, just found this out:

> Before npm 7 yarn.lock was ignored by npm, but this is no longer the case. It can now use it to keep itself up to date with the package tree.